cves/CVE-2017-9798.yml

```diff
@@ -121,10 +126,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
-  automated:
-  google:
+  answer: |
+    This vulnerability was found by two people. Hanno Böck on September 18, 2017 and
+    Jon King of OPNAV N1, Giulio Comi of Horizon Security on Oct 16, 2018 
```

Comment:
Is there any mention how this vulnerability was found?

Reply:
There was no mention.